pkg/schema/di.go

```diff
@@ -129,7 +172,8 @@ type CoreRegistry = schemaRegistry
 // because referencing these in other Go code will likely trigger component
 // group condition rules
 type schemaRegistry struct {
-	M sync.Map
+	M             sync.Map
+	schemeBuilder *scheme.Builder
```

Comment:
What's the thinking behind having this here, rather than assembling it over in the bridge?